### ISSUE TITLE
Restrict cdr.co

### ISSUE THREAD

**Issue by Niehaus1301** [codercom/features#175]
**Is your feature request related to a problem? Please describe.**
My cdr.co link is public accessible. This could be a security risk.

**Describe the solution you'd like**
Add an option to restrict access to IP address (probably the IP address which is used to access the IDE?)

**Describe alternatives you've considered**
Password authentication (and of course you should still be able to allow public access)

**Comment by nhooyr** [comment]
You are correct that it is publicly accessible but given the url is randomly generated, its very very unlikely someone comes across your very project.
